task/buildah-remote/0.1/buildah-remote.yaml

```diff
@@ -189,6 +189,8 @@ spec:
       fi
 
       rsync -ra $(workspaces.source.path)/ "$SSH_HOST:$BUILD_DIR/workspaces/source/"
+      rsync -ra /entitlement/ "$SSH_HOST:$BUILD_DIR/volumes/etc-pki-entitlement/"
+      rsync -ra /mnt/trusted-ca/ "$SSH_HOST:$BUILD_DIR/volumes/trusted-ca/"
```

Comment:
Is it intentional that `buildah-remote` mounts the trusted-ca but `buildah-remote-oci-ta` doesn't?

Reply:
All of these changes are generated by changes in the generator in the multi-platform controller. 

We should probably consider moving that code to here, so that it is all in the right place. 

In this case the reason appears to be that the trusted-ca mount is missing from buildah-oci-ta, so the generator matches the original source.